style.css

```diff
@@ -0,0 +1,18 @@
+.additional-text {
```

Comment:
I think it needs border same as wire-frame, because i think we have to work depend on the wire-frame. `.additional-text {border: solid  3px  black}`. And do you think it needs background-color ?

Reply:
@Feruzteame the background color and the border are not a must, it is just matter of creativity. I agree with @harunaltunhr we dont have to add the border. and yes @murvanessa I asked Harun to make aside of 30% and mine is 65%

Reply:
good idea